src/VirtualList/VirtualList.js

```diff
@@ -237,11 +241,19 @@ class VirtualList extends React.Component {
       Object.keys(VirtualList.propTypes)
     );
 
+    const safeRenderBufferItem = styles => {
```

Comment:
I wonder we do we need an ad-hoc function here 🤔 Can't we declare it outside of the render method?

Reply:
private package function?